src/default_env_vars.yml

```diff
@@ -1,12 +1,12 @@
-USING_PICKLE: false
-DEBUG: false
-DOWNLOAD_HOST: "download.alliancegenome.org"
-GENERATE_REPORTS: false
+USING_PICKLE: False
+DEBUG: False
+DOWNLOAD_HOST: "downloaddev.alliancegenome.org"
```

Comment:
Could we talk about why we want the default to be downloaddev?  I would prefer the default be download and similarly fms vs fmsdev.

Reply:
The reason for this is if a develop downloads this code and runs it, it WON'T hit production.

Reply:
These can be passed either via docker compose or via the command line... that way its intentional in order to hit production.

Reply:
Can you clarify?  I'd rather the loader get files from download.alliancegenome.org rather than downloaddev.alliancegenome.org because downloaddev.alliancegenome.org does not usually have the most up to date files in it.  "hitting" production meaning uploading to production FMS?  (vs. just downloading from FMS...maybe we need two variables)

Reply:
yes... I agree if all the loader did was download files. However the gene descriptions get uploaded to the FMS.

Reply:
so I suppose the download.alliancegenome.org could be left as is... but fms shouldn't be.

Reply:
agree - though gene descriptions uses another parameter already "GENERATE_REPORTS" which is set to false by default.

Reply:
I do use fms.alliancegenome.org to get the download path -- which is not always the same on fmsdev.

I would just prefer if this change had some time to think it out - I think it should be removed from this PR for now.  If we add another variable to make this work, then we need changes to gene descriptions in order to use it, if we change to downloaddev and fmsdev, then we might need devops changes (like adding a job somewhere that I can run myself on demand) to get downloaddev and fmsdev in sync with download and fms.